src/app/wizard/services/app-launcher-project-progress.service.ts

```diff
@@ -17,8 +17,12 @@ export class AppLauncherProjectProgressService implements ProjectProgressService
   constructor(private helperService: HelperService) {
     this.END_POINT = this.helperService.getBackendUrl();
     this.END_POINT = this.END_POINT.split('/api')[0];
-
-    if (this.END_POINT && (this.END_POINT.startsWith("/") || this.END_POINT.startsWith(":"))) {
+    if (this.END_POINT.indexOf('https') !== -1) {
+      this.END_POINT = this.END_POINT.replace('https', 'wss');
+    } else if (this.END_POINT.indexOf('http') !== -1) {
+      this.END_POINT = this.END_POINT.replace('http', 'wss');
```

Comment:
I think this is incorrect, it should be `ws`, not `wss`

Reply:
Yep @gastaldi already fixed that you could have reverted my last commit

Reply:
Aha indeed, I hadn't seen the later commit